<test-history test='Basic 4 (c2s)'>
<OUTPUT test='Basic 4 (c2s)'>
<![CDATA[<stream:stream xmlns='jabber:client' xmlns:stream='http://etherx.jabber.org/streams' to='test-d' version='1.0'/>
]]></OUTPUT>
<INPUT test='Basic 4 (c2s)'>
<![CDATA[<stream:stream xmlns="jabber:client" from="127.0.0.1" version="1.0" xmlns:stream="http://etherx.jabber.org/streams" id="tigase-error-tigase" xml:lang="en"/>
]]></INPUT>
<OUTPUT test='Basic 4 (c2s)'>
<![CDATA[<iq type='get' id='all-xmpp-test_1' to='test-d'><query xmlns='jabber:iq:auth'><username>all-xmpp-test</username></query></iq>
]]></OUTPUT>
<INPUT test='Basic 4 (c2s)'>
<![CDATA[tigase.test.impl.TestAuth, all-xmpp-test@test-d/def-user-resource(all-xmpp-test-password), Software caused connection abort: recv failed
]]></INPUT>

</test-history>